Templates/TraditionalBridge/InterfaceMethodDispatchers.tpl

```diff
@@ -459,7 +457,7 @@ namespace <NAMESPACE>.<INTERFACE_NAME>
 ;//Single structure processing
 ;//
             ;;Argument <COUNTER_1_VALUE>: Single <ParameterStructureNoplural> record
-            serializer.ArgumentData(<COUNTER_1_VALUE>,arg<COUNTER_1_VALUE>,FieldDataType.DataObjectField,<PARAMETER_SIZE>,"<PARAMETER_STRUCTURE>",false)
+            serializer.ArgumentData(<COUNTER_1_VALUE>,arg<COUNTER_1_VALUE>,FieldDataType.DataObjectField,<PARAMETER_SIZE>,"<PARAMETER_STRUCTURE>",true)
```

Comment:
we have a codegen expression to determine if a structure contains binary data, should it be used here rather than hard coding to always encode binary?

Reply:
cannot solve this issue